infrastructure/terraform.test.tfvars

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2023, Xgrid Inc, https://xgrid.co
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#        http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+namespace      = "test"
+region         = "eu-west-1"
+account_id     = "123456789012"
+vpc_cidr_block = "15.1.1.0/24"
+public_subnet_cidr_block = {
+  "eu-west-1a" = "15.1.1.0/26"
+  "eu-west-1c" = "15.1.1.64/26"
+}
+domain_name                = "test.xgrid.co"
+hosted_zone_id             = "Z053166920YP1STI0EK5X"
```

Comment:
if it is valid domain name please remove it

Reply:
updated in e3bef34deca5120d06a3147f4c52cfbea7e2016e